mdoc/Mono.Documentation/Updater/Formatters/MemberFormatter.cs

```diff
@@ -390,13 +390,32 @@ private bool IsTypeOfOperatorWithUnboundGenericType(TypeReference type, IEnumera
             return false;
         }
 
-        private void AppendGenericTypeParameter (StringBuilder buf, TypeReference type, IAttributeParserContext context, bool useTypeProjection = true)
+        private bool IsGenericWithValueTypeArg(GenericInstanceType type)
+        {
+            if (type.Name == "Nullable`1" && type.HasGenericArguments && type.GenericArguments[0].IsValueType)
```

Comment:
please help to check if we can directly use  **type.IsValueType**  instead of **type.HasGenericArguments && type.GenericArguments[0].IsValueType**

Reply:
Since **public struct Nullable<T> where T : struct** , so we can just only check "type.Name == "Nullable`1"